react/Viewer/Panel/Qualification.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+
+import { useQuery, models, getReferencedBy } from 'cozy-client'
+
+import List from '../../MuiCozyTheme/List'
+import ListItem from '../../MuiCozyTheme/ListItem'
+import ListItemText from '../../ListItemText'
+import Typography from '../../Typography'
+import { withViewerLocales } from '../withViewerLocales'
+import { getContactByIds } from '../queries'
+
+const {
+  contact: { getFullname },
+  document: {
+    locales: { getBoundT }
+  }
+} = models
+
+const Qualification = ({ file = {}, t, f, lang }) => {
+  const scannerT = getBoundT(lang)
+
+  const { name: filename, metadata = {} } = file
+  const {
+    qualification = {},
+    page: pageLabel,
+    datetime,
+    datetimeLabel
+  } = metadata
+
+  const contactIds = getReferencedBy(file, 'io.cozy.contacts').map(
+    ref => ref.id
+  )
+
+  const buildContactDef = getContactByIds(contactIds)
```

Comment:
peut-être renommer getContactByIds() par buildContactByIdsQuery() (voir exemple dans queries.js de Drive). Ce qui donnerait `const contactByIdsQuery = buildContactByIdsQuery(contactIds)`. Le getXxxx fait plutôt penser qu'on fait un fetch dans la méthode.

Reply:
> Le getXxxx fait plutôt penser qu'on fait un fetch dans la méthode.

https://github.com/cozy/cozy-guidelines#naming-of-functions. 

> getSomething : should access data without making a server side call

No fetch involved.

Reply:
exact c'est fetchSomething !